lxd/certificates.go

```diff
@@ -528,7 +544,15 @@ func certificatesPost(d *Daemon, r *http.Request) response.Response {
 	}
 
 	// Handle requests by non-admin users.
-	if !s.Authorizer.UserIsAdmin(r) {
+	var userCanCreateCertificates bool
```

Comment:
I think this section could do with a comment explaining whats going on here and why its special.